backend/crypto.cpp

```diff
@@ -2,41 +2,65 @@
 #include "modes.h"
 #include "aes.h"
 #include "files.h"
+#include "osrng.h"
+#include <ios>
+#include <fstream>
 
 
 int encrypt_file(char const* in_file, char const* out_file, char const* key_file)
 {
+    // open output
+    std::ofstream fout(out_file, std::ios::binary);
```

Comment:
need to check if fout was successful  (only do further stuff if file has been opened without problem)